charter/index.html

```diff
@@ -238,44 +238,62 @@ <h3>
                         </p>
-                            <ol>
-                                <li>
-                                    <a href="https://openid.net/connect/">OpenID Connect</a>
-                                </li>
-                                <li>
-                                    Access management and data integrity using <a href="https://www.w3.org/TR/vc-data-model/">Verifiable Credentials Data Model</a>
-                                </li>
-                                <li>
-                                    Identity mechanisms such as <a href="https://www.w3.org/2005/Incubator/webid/spec/identity/">WebID</a> and <a href="https://www.w3.org/TR/did-core/">DID</a>
-                                </li>
-                                <li>
-                                    Notifications about resource changes using <a href="https://solidproject.org/TR/notifications-protocol">Solid Notifications Protocol</a>
-                                </li>
-                                <li>
-                                    Authorization mechanisms such as <a href="https://solidproject.org/TR/wac">Web Access Control</a> and <a href="https://solidproject.org/TR/acp">Access Control Policy</a>
-                                </li>
-                            </ol>
                     </dd>
                 </dl>
 
                 <p>
-                    Note that the WG may decide, based on editorial and readability considerations, to spin off sections into separate Recommendations track specifications.
-                </p>
-                <p>
-                    All specifications, regardless of their progress along the W3C process line from Working Draft to Recommendation, will be given a version number, such as 0.9, which will be incremented like from 0.9 to 0.10 for minor changes. A major version increment like 2.2 to 3.0 will be used if there is an incompatible change in the protocol. The required versions of each dependency will be given in each spec.
+                    All specifications, regardless of their progress along the W3C Process line from First Public Working Draft to Recommendation, will indicate their version using the <a href="https://semver.org/">Semantic Versioning</a> scheme. The required versions of each dependency will be given in each specification.
                 </p>
         </section>
 
+        <section id="dependencies">
+          <h3>Dependencies</h3>
+
+          <p>Depending on the W3C Solid Community Group progress, including consideration for <a href="https://www.w3.org/2021/Process-20211102/#adequate-implementation">adequate implementation experience</a>, the Group will also produce W3C Recommendations for the following documents:</p>
```

Comment:
Based on the conversation during the meeting I don't think this sentence still doesn't give clarity on how the WG plans to deal with normative references (hard dependencies). Given that for the spec to reach REC all the normative references have to be at most one maturity level behind if Solid Protocol has a hard dependency on any of the specs below, their delivery IMO can not depend on Solid CG progress. I think we need to decide on one of the two options:

* Solid Protocol will have a hard dependency on **all** of the specs below and all those will have to be delivered independently from CG progress.
* Solid Protocol can still be delivered **without** having a hard dependency on **any** of the specs below, in this case, the WG doesn't need to ensure that all of them will be delivered.

Possibly WG can commit to delivering Solid-OIDC, WAC & ACP and have them as normative dependencies/references in the Solid Protocol. Notifications could be initially proposed as an informative dependency/reference. If WG manages to get notifications to a sufficient maturity level it could also become a normative dependency.

Reply:
Normative references in Solid Protocol, Version 0.10.0, that do not have a FPWD or later status at W3C:
* Solid-OIDC (Solid CG)
* WAC (Solid CG)
* ACP (Solid CG)
* Solid Notifications Protocol (Solid CG)
* WebID (WebID CG)
* N3 (Notation 3 CG)

As per Version 0.10.0, the Solid WG will have to work with the Solid, WebID, and Notation 3 CGs to ensure that those normative references are progressed as well.

Similarly, some of the normatively referenced IETF specifications may need to have a more mature status.

The mention of Solid CG was primarily about what's generally under the purview of the Solid CG, and not particularly about waiting for the Solid CG to deliver. If folks are so keen to have those work items mature, there has been absolutely nothing, and I mean nothing posed as a barrier in the CG. And, if so desired, they can be a deliverable of the WG pretty much overnight. It is evident that the work that needs to be done is a matter of commitment. If any editor/author of a CG work items is not committed, engaging, we can assign those that are committed. Including proper reviewing..

I have no objection to listing all of those normative references as deliverables but I suggest that we do not until requested by W3C.

That will give us some flexibility, if so needed, to adopt the Solid Protocol, and the declare new deliverables that accurately reflects it.